benchmark/fio_benchmark_setup.yml

```diff
@@ -66,12 +67,10 @@
         - /opt/ceph_bench/logs
         - /mnt/fio_bench
     - name: Template out fio_bench files
-      config_template:
-        src: "templates/{{ item.src }}"
-        dest: "/opt/ceph_bench/{{ item.name }}.cfg"
-        config_overrides: "{{ item.override }}"
-        config_type: "ini"
-      with_items: "{{ fio_bench_list }}"
+      template:
```

Comment:
config_template would allow us to override settings on existing tests, we use this in testing:

https://github.com/rcbops/rpc-ceph/blob/master/tests/test-vars.yml#L86 - setting size and runtime etc which is really useful.
It can also be used on live systems to override any additional settings or set new ones.

Reply:
We could probably still use config_template here.
Something like:
config_template:
  src: fio.cfg.j2
  dest: /opt/ceph_bench/{{ item.settings.key }}.cfg
  config_overrides: "{{ item.override }}"
  config_type: ini
with_items: "{{ fio_bench_list }}"

fio_bench_list:
  bench_settings:
     settings... etc.
   overrides: {{ bench_overrides }}

Then in the template do something like:

{% for section, vals in item.settings.value.items() %}
[{{ section }}]
{% for k, v in vals.items() %}
{{ k }}={{ v }}
{% endfor %}
{% endfor %}